### ISSUE TITLE
Commented modules still included

### ISSUE THREAD

**Issue by dmitriz** [yagoferrer/gulp-angular-modules#4]
When I comment a module:

``` js
// angular.module('commented', []);
```

it is included by the plugin nevertheless.

Further, it seems only the first `module` in the file is recognized by the plugin.
So if I have

``` js
// angular.module('oldmodule', []) ...
angular.module('newmodule', []) ...
```

then `oldmodule` is taken by the plugin instead of `newmodule`.

**Comment by yagoferrer** [comment]
Thank you @dmitriz. I'm fixing the bug

**Comment by yagoferrer** [comment]
I released 0.1.8 with the bug fix.
